x-pack/legacy/plugins/reporting/public/panel_actions/get_csv_panel_action.tsx

```diff
@@ -68,7 +68,7 @@ class GetCsvReportPanelAction implements IAction<ActionContext> {
       return {};
     }
 
-    return searchEmbeddable.searchScope.searchSource.getSearchRequestBody();
+    return searchEmbeddable.savedSearch.searchSource.getSearchRequestBody();
```

Comment:
@stacey-gammon @lukasolson I tagged you as reviewers to confirm this is the correct way to get the request data, from the Embeddable API perspective

Reply:
I don't believe there is any better way to get the search request body from embeddable api in a generic fashion.  There is the inspector on embeddables, maybe you could get it this way? But csv download is specific to SearchEmbeddable, so this probably works fine, as long as `savedSearch` is marked as public in that interface.

Reply:
It hopefully will work fine, but, `savedSearch` is designed to be private: https://github.com/elastic/kibana/blob/c28ce03/src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable.ts#L99 per the class. I'm not sure how Typescript allowed that.